tool/lib/commands/analyze.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+import 'package:cli_util/cli_logging.dart';
+
+import '../model.dart';
+import '../utils.dart';
+
+class AnalyzeCommand extends Command {
+  @override
+  String get name => 'analyze';
+
+  @override
+  String get description => 'Analyze all DevTools packages.';
+
+  @override
+  Future run() async {
+    final FlutterSdk sdk = FlutterSdk.getSdk();
```

Comment:
remove type

Here and elsewhere in this method

Reply:
done, and removed in several more places in this PR